pkgs/development/python-modules/unicode-slugify/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, nose
+, six
+, unittest2
+, unidecode
+}:
+
+buildPythonPackage rec {
+  pname = "unicode-slugify";
+  version = "0.1.3";
+
+  propagatedBuildInputs = [ unidecode ];
+
+  checkInputs = [ nose six unittest2 ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "34cf3afefa6480efe705a4fc0eaeeaf7f49754aec322ba3e8b2f27dc1cbcf650";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Unicode Slugify is a slugifier that generates unicode slugs. It was originally used in the Firefox Add-ons web site to generate slugs for add-ons and add-on collections. Many of these add-ons and collections had unicode characters and required more than simple transliteration.";
```

Comment:
This description is too long. Can we shorten it and not use a period at the end?

Reply:
fixed